site/layouts/shortcodes/nic-operator-version.html

```diff
@@ -1 +1 @@
-2.4.2
\ No newline at end of file
+2.5.0
```

Comment:
should this be `3.0.0` ? as its based on helm

Reply:
updated 👍